lib/gitarro/backend.rb

```diff
@@ -63,11 +64,16 @@ def initialize(option = nil)
     @gbexec = TestExecutor.new(@options)
   end
 
-  # public method for get prs opens
-  # given a repo
-  def open_prs
-    prs = @client.pull_requests(@repo, state: 'open')
-    puts 'no Pull request OPEN on the REPO!' unless prs.any?
+  # public method for get prs opened and matching the change_newer
+  # condition
+  def open_newer_prs
+    prs = []
+    @client.pull_requests(@repo, state: 'open').each do |pr|
+      if Time.now.utc - pr.updated_at < @options[:change_newer] * 60 ||
```

Comment:
i think we need here a private method for 2 reasons:
something like:

```pr_comment_commit_changed_less_then(min)```
```console
 return true  if Time.now.utc - pr.updated_at < @options[:change_newer] * 60 || ..
```
-  for making the code more abstract from the algorithm itself, and more maintenable.
- it will remove the  the if and end, making a inline if (more compact)

Reply:
Ok, but maybe pr_changed_less_than(pr, min) will be a easier name. What do you think?

Reply:
Yop, but if so i would make a comment above the function saying what trigger tje change